docs/pages/inboxes/history-sync.mdx

```diff
@@ -33,26 +34,34 @@ You can choose to [run your own server](https://github.com/xmtp/xmtp-message-his
 When your app initializes an XMTP client and a `historySyncUrl` client option is present, history sync automatically triggers an initial sync request and creates an encrypted payload.
 
 <div>
-    <img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/refs/heads/main/docs/pages/img/history-sync-step1.png" width="600px" />
+    <img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/refs/heads/main/docs/pages/img/history-sync-1-state.png" width="600px" />
 </div>
 
-History sync then uploads the payload, sends a sync reply, and pulls all conversation state history into the new app installation, merging it with the existing app installations in the sync group.
+History sync then uploads the payload, sends a sync reply, and pulls all conversation state history into the new app installation, merging it with the existing app installations in the installation state group.
 
 <div>
-    <img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/refs/heads/main/docs/pages/img/history-sync-step2.png" width="600px" />
+    <img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/refs/heads/main/docs/pages/img/history-sync-2-state.png" width="600px" />
 </div>
 
-Ongoing updates to history are streamed automatically. Updates, whether for user consent preferences or messages, are sent across the sync group, ensuring all app installations have up-to-date information.
+Ongoing updates to history are streamed automatically. Updates, whether for user consent preferences or messages, are sent across the installation state group, ensuring all app installations have up-to-date information.
 
 History syncs are accomplished using these components:
 
-- Sync group
+- Installation state group
 - Sync worker
 - History server
 
-### Sync group
+### Installation state group
 
-A sync group is a special [Messaging Layer Security](/protocol/security) group that includes all of the user’s devices. The sync group is used to send serialized sync messages across the user's devices. The sync group is filtered out of most queries by default so they don’t appear in the user’s inbox.
+An installation state group is a special [Messaging Layer Security](/protocol/security) group that includes all of the user's installations (devices). The installation state group is used to send serialized sync messages of a local installation state across a user's other installations. This state information includes:
```

Comment:
Sometimes we use "installation" and "device" interchangeably. I prefer using "installation" because it is more precise and accurate - is that right? Like in the protocol - we call it an installation - not a device, is that right?

Reply:
Yeah, I think it's a good idea to stick to the technical term "installation" too. I find myself going back and forth because it's easier to say and type "Device"

Reply:
Thank you for discussing installation = device with me, Coda. I think we are on the same page now and I'll incorporate updates in this PR!